overlord/snapstate/refresh_test.go

```diff
@@ -20,44 +20,28 @@
 package snapstate_test
 
 import (
-	"bytes"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-
 	. "gopkg.in/check.v1"
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/overlord/snapstate"
 	"github.com/snapcore/snapd/overlord/state"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/snaptest"
-	"github.com/snapcore/snapd/testutil"
 )
 
 type refreshSuite struct {
-	testutil.BaseTest
```

Comment:
why remove testutil.BaseTest ? if you leave that in, you could simplify the suite to use `AddCleanup` instead of the TearDownTest method

Reply:
yes, we are generally trying to use testutil.BaseTest more, not less, unless this some kind of merge artificat

Reply:
Sure thing, adjusting now. This code is pretty old so it's good to pick up things like that during review and adjust them as necessary.

Reply:
Done